api-admin/Dockerfile

```
@@ -0,0 +1,8 @@
+# 1. Java 21이 들어있는 공식 경량 이미지를 사용
+FROM eclipse-temurin:21-jdk-alpine
+
+# 2. 빌드 산출물(jar) 복사
+COPY api-admin/build/libs/*.jar app.jar
```

Hadolint (2.12.0)

[warning] `COPY` to a relative destination without `WORKDIR` set.

(DL3045)